src/idiomatic/foundations-api-design/predictable-api/naming-conventions/09-owned.md

```diff
@@ -0,0 +1,21 @@
+---
+minutes: 2
+---
+
+# Owned
+
+Suffix for functions and methods about owned values in a context where
+references are common.
+
+```
+String::to_owned
+Cow::into_owned
+```
+
+<details>
+- Most often seen in to_owned methods, especially on strings.
```

Comment:
I believe all of the standard library "to_owned" methods come from the blanket implementation of the ToOwned trait. I'm not contradicting what you wrote, just noting that the speaker notes seem to imply that there are many (or at least multiple) "to_owned" methods, when in fact there is just one, available on multiple types of course.